__tests__/app.test.js

```diff
@@ -190,4 +190,47 @@ describe("GET /api/articles/:article_id/comments",() => {
             expect(body.message).toBe('Path not found')
         })
     })
+})
+
+describe("POST /api/articles/:article_id/comments",()=>{
+    test("POST: 201 inserts a new comment to the db and responds with the posted comment back to the user",() => {
+        const newComment = {
+            "username" : "butter_bridge",
+            "body"  : "The article have given me some beautiful insights into the world of ecommerce"
+        }
+        return request(app)
+        .post('/api/articles/2/comments')
+        .send(newComment)
+        .expect(201)
+        .then(({body})=>{
+            expect(body.comment.comment_id).toBe(19)
+            expect(body.comment.author).toBe('butter_bridge')
+            expect(body.comment.body).toBe("The article have given me some beautiful insights into the world of ecommerce")
+        })
```

Comment:
is there anything else you would expect to see on the comment? For example, the article_id?